bin/cli.js

```diff
@@ -29,17 +37,35 @@ function getParentBranches() {
 }
 
 function bump(type) {
-    const publishedVersions = JSON.stringify(execSync(`npm show . versions -json`, {encoding: 'utf8'}));
     let newVersion;
-    do {
-        newVersion = execSync(`npm version ${type}`, {encoding: 'utf8'}).trim().substr(1);
-    } while(publishedVersions.indexOf(newVersion) !== -1);
+
+    if (fs.existsSync('pom.xml')) {
+        const publishedVersion = execSync(`mvn -q -Dexec.executable="echo" -Dexec.args='\${project.version}' --non-recursive exec:exec`, {encoding: 'utf8'}).trim();
```

Comment:
You might have to call `toString()` before trim since execSync sometimes return a buffer